### DISCUSSION TITLE
Deploying Qdrant in a multi-cluster environment

### DISCUSSION THREAD

**Discussion by robertmacyiii** [qdrant/qdrant#3939] [Q&A] [answered]
I am curious if any community members have tried or thought through how Qdrant could be deployed in a multi-cluster environment on Kubernetes. I understand high availability can be achieved through a  distributed deployment setup on a single cluster, but how would you approach a multi-cluster setup where you want to keep the collections up to date across all clusters? Would you try to have cross-cluster communication or restrict writes to a single cluster and have the other clusters act solely as read replicas? I'm new to this space so any insights are greatly appreciated!

**Comment by timvisee** [comment, marked as answer]
Though we don't have any instruments for multi-region deployments and specialized nodes at this time, it is on our roadmap for this year: https://github.com/qdrant/qdrant/blob/master/docs/roadmap/README.md#core-milestones

You can simply create a collection on a multi region cluster, but latency across regions probably badly affects performance in its current stage.

At this time you're possibly better off using separate clusters in separate regions.

**Reply by buildgreatthings** [reply]
Hi @timvisee! Since we're hitting December, is there any update to this?

**Reply by osehmathias** [reply]
It's really not that hard to set up replicas across regions. Just make sure you have cross-region VPC peering set up in AWS. Also, you can use listener nodes to improve performance if you need to.

**Comment by osehmathias** [comment]
You can use this to create a multiregion distributed cluster on AWS. The key is the peering configuration. 

https://github.com/duohub-ai/qdrant-multiregion-distribution
